src/Features/Core/Portable/Copilot/ImplementationDetails.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using Microsoft.CodeAnalysis.CSharp.Syntax;
```

Comment:
this is not ok.  you are referencing C# specific code in a common library.  Note: i'm actually surrpised this compiled.